three_dimension_array_with_shape_fill_0_to_255.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def three_dimension_array_with_shape_fill_0_to_255():
+    """
+    
+    :return:
+    """
+    np.random.seed(32)
```

Comment:
Why is the seed inside the function, if you call the function multiple times inside the program you will ge the same results.